crud/common/utils.lua

```diff
@@ -14,6 +14,8 @@ local ParseOperationsError = errors.new_class('ParseOperationsError', {capture_s
 local ShardingError = errors.new_class('ShardingError', {capture_stack = false})
 local GetSpaceFormatError = errors.new_class('GetSpaceFormatError', {capture_stack = false})
 local FilterFieldsError = errors.new_class('FilterFieldsError', {capture_stack = false})
+local GetReplicaStateError = errors.new_class('GetStorageStateError')
+local fiber_clock = require('fiber').clock
```

Comment:
If `fiber.time()` calls are splitted by a yield, it should be okay, I guess.

Reply:
According to doc, looks like it is recommended to use fiber.clock for timeouts because it is monotonic and returns elapsed wall-clock time. As I can see, it is implemented using _ev_monotonic_now_ call and is also used for timeouts in tarantool source code.